testData/typeAlias/unionTypeParams.d.ts

```diff
@@ -0,0 +1,14 @@
+type Values<V> = List<V>;
+type ListOrMultiMap<K,V> = List<V> | Map<K,Values<V>>;
```

Comment:
It seems difficult to find the right balance between using Kotlin's alias type and dealing with TS' union type.  TS aliases can contain unions.  I'd prefer to defer that for now for that reason.